source/topologies/rs-to-sharded.txt

```diff
@@ -0,0 +1,115 @@
+.. _c2c-rs-to-sharded:
+
+=======================================
+Sync a Replica Set to a Sharded Cluster
+=======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+This page outlines the procedure to synchronize data from a replica set to a 
+sharded cluster using :program:`mongosync`. 
+
+.. note::
+
+   ``mongosync`` doesn't support sync from a sharded cluster to a replica set.
+
+To perform a sync between two replica sets, see :ref:`c2c-quickstart`.
+
+Before You Begin
+----------------
+
+Before you begin your sync:
+
+- Define your source replica set.
+- Define your destination sharded cluster.
+- Define an administrative user that has permissions in both clusters.
+
+You must also download ``mongosync``. To learn more, see 
+the :ref:`mongosync setup <c2c-quickstart-setup>` instructions.
+
+Steps 
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: Connect the replica set and sharded cluster. 
```

Comment:
No punctuation for step titles, this applies globally to this edit. 

```suggestion
   .. step:: Connect the replica set and sharded cluster
```

Reply:
Good catch, I will keep that in mind from now on!